flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -52,13 +57,19 @@
 
   // Can not just use "type", it conflicts with CATALOG_TYPE.
   public static final String ICEBERG_CATALOG_TYPE = "catalog-type";
+
   public static final String HIVE_URI = "uri";
   public static final String HIVE_CLIENT_POOL_SIZE = "clients";
   public static final String HADOOP_WAREHOUSE_LOCATION = "warehouse";
 
+  public static final String HIVE_SITE_PATH = "hive-site-path";
+  public static final String HIVE_SITE_SCHEME_FILE = "file";
```

Comment:
If we plan to support hadoop path ,  would we still need the `file` or `hdfs` schema ?   Just load those configurations files by hadoop filesystem ?

Reply:
I have tested this, if we specify a remote path, as the following code . 
```
    Configuration conf = new Configuration();   
    Path path = new Path("hdfs://localhost/data/flink/conf/hive-site.xml");
    conf.addResource(path);
    String warehouseLocation = conf.get("hive.metastore.warehouse.dir");
    System.out.println(warehouseLocation);
```
It will not be able to get the attributes in hive-site.xml, the result is null.

If you specify a local path, 
```
 Path path = new Path("file:///tmp/hive-site.xml");
```
we can get it, so I use this scheme to make some judgments. If it is a non-local path, download it to the local first, and then load it.